core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-category-list/dot-category-field-category-list.component.ts

```diff
@@ -89,12 +90,15 @@ export class DotCategoryFieldCategoryListComponent implements AfterViewInit {
      */
     itemsSelected: string[];
 
+    #cdr = inject(ChangeDetectorRef);
+
     readonly #destroyRef = inject(DestroyRef);
 
     readonly #effectRef = effect(() => {
-        // Todo: find a better way to update this
+        // Todo: change itemsSelected to use model when update Angular to >17.3
         // Initial selected items from the contentlet
         this.itemsSelected = this.selected();
+        this.#cdr.markForCheck(); // force refresh
```

Comment:
why this is needed?

Reply:
The effect is triggered by the change of selected input, but the model (in this case, not a signal) is assigned to remove the checked checkbox; that is why I left a comment there to change it to model later.